server/controllers/pages.js

```diff
@@ -16,7 +16,27 @@ const entryHelper = require('../helpers/entry')
  * Edit document in Markdown
  */
 router.get('/edit/*', (req, res, next) => {
-  if (!res.locals.rights.write) {
+  var xPaths = ['news', 'general'] // the final word in the url For.e.g : news in /edit/news, general in /edit/general 
```

Comment:
I believe this array should be read from the config.yml file so that it can be changed without needing to make any commits. This also considering we're unlikely to be giving all this back to upstream.